composer.lock

```diff
@@ -196,13 +196,13 @@
                 "reference": "bfdd976",
                 "shasum": ""
             },
-            "require": {
-                "php": ">=5.3.2"
-            },
+			"require": {
+			  "php": ">=5.3.2"
+			},
             "require-dev": {
                 "codacy/coverage": "^1.4",
-                "phpunit/phpunit": "^4.8.36"
-            },
+				"phpunit/phpunit": "~4.8"
+			},
```

Comment:
I've manually reverted the psr-4 path to `lib/Sabberworm/CSS/` in 1c908500d6b897d0d73479b50fce54225c41d499. This seems like a Composer bug so I'll report the bug.

Reply:
See https://github.com/composer/composer/issues/9990.

Reply:
Turns out the behavior is as intended, as explained in https://github.com/composer/composer/issues/9990#issuecomment-875398763. I'm not sure how to resolve this then since installing the dependency from the `master` branch is essentially broken now. Maybe we could fork it in the interim until a new release is made?

Reply:
Fork it in order to create a release tag to reference?

I hope that a new release of PHP-CSS-Parser will be made soon with all of the improvements being made recently.

Reply:
I made a Composer plugin as a method of patching the `composer.lock` file as well, but I'm not sure if we need to employ that as yet.

Reply:
😎  You're a composer whisperer.

Reply:
Opened a PR to add it: https://github.com/ampproject/amp-wp/pull/6464.